include/a3/ui_f/hpp/defineDIKCodes.inc

```diff
@@ -1,3 +1,6 @@
+#pragma hemtt suppress pw3_padded_arg file
+#pragma hemtt suppress pw3_padded_arg config
+
```

Comment:
@BrettMayson - still getting with either/both line, not sure if error or just how the includes work
```
[PW3] Warning: padding a macro argument
     ╭─[/a3/ui_f/hpp/defineDIKCodes.inc:119:36]
     │
 119 │ #define DIK_NUMPADENTER     0x9C    /* Enter on numeric keypad */
     │                                    ┬
     │                                    ╰── padding a macro argument is likely unintended
```

Reply:
It's might be where DIK_NUMPADENTER is being used, and the error is just pointing to the wrong location. The comment will be removed, but the padded whitespace won't

Reply:
yeah, it's on use not on define
which is actually good because it means we don't have to touch the bi files
a nice-to-have would be to indicate usage location